src/com/sun/ts/tests/signaturetest/SigTest.java

```diff
@@ -51,8 +52,12 @@
   protected SignatureTestDriver getSigTestDriver() {
 
     if (driver == null) {
-      driver = SignatureTestDriverFactory
-          .getInstance(SignatureTestDriverFactory.SIG_TEST);
+      String version = (String) System.getProperties().get("java.version");
```

Comment:
Thanks. Hope you were able to test this change and run sig tests in jdk11.

Reply:
Yes. I ran the tests with both Java 8 and Java 11 with this PR in place.